packages/controllers/organization_detail.coffee

```diff
@@ -5,7 +5,36 @@ if Meteor.isClient
   Template.organizationDetail.helpers
     organization: ->
       Organizations.findOne(@organizationId)
+    userIsMember: ->
+      UserProfiles.findOne({
+        userId: Meteor.userId(),
+        memberOfOrgs: @organizationId
+      })
+    memberCount: ->
+      Organizations.findOne(@organizationId)?.getMemberProfiles().fetch().length
+    members: ->
+      Organizations.findOne(@organizationId)?.getMemberProfiles()
+
+  Template.organizationDetail.events
+    'click .join-organization' : (event, instance) ->
+      orgId = instance.data.organizationId
+      Meteor.call 'joinOrganization', orgId, (error, response) ->
+        if error
+          toastr.error("Error")
+        else
+          toastr.success("Success")
 
 if Meteor.isServer
-  Meteor.publish 'organizationDetail', (id) ->
-    Organizations.find(id)
+
+  Meteor.publish('organizationDetail', (id) ->
+    [
+      Organizations.find(id)
+      UserProfiles.find({
+        memberOfOrgs: id
+      })
```

Comment:
It turns out selectively hiding the emails is a pretty difficult thing to do. The approach used for profile detail publication hides the email of only a single user, so Mongo's `fields` property can be used to hide the field, however when there are multiple users, some with hidden emails and some without, `fields` property cannot be used because it will need different values for each document in the collection.

I tried publishing multiple cursors for the UserProfiles collection like this, but Meteor does not support returning multiple cursors for the same collection:

```
[
      UserProfiles.find({
        memberOfOrgs: id
        emailHidden: false
      }),
      UserProfiles.find({
        memberOfOrgs: id
        emailHidden: true
      }, {
        fields:
          emailAddress: false
      })
]
```
There is a library that does this, however it has an explicit warning that it is not ready for production: https://atmospherejs.com/mrt/smart-publish

I think publish-composite might be able to handle this situation, however we would need to resolve it's Astronomy incompatibilities.

We could use the Meteor.publish added/changed/removed interface, although the code will be pretty complicated in that case. Here's an example of what the code might look like: https://gist.github.com/colllin/8321227

I'm leaning towards using the added/changed/removed interface. Do you have a preference for a particular approach or any other ideas on how we could accomplish this?